### ISSUE TITLE
powershell not working (arch)

### ISSUE THREAD

**Issue by Phelsong** [warpdotdev/Warp#5624]
### Dupe Check

- [X] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABugs) and there are no duplicates

### Describe the bug

When trying to launch pwsh warp hangs with the message "starting powershell"

this issue has been ongoing since the support announcement. I have 4 machines I can reproduce this on. I had it working on Ubuntu at one point for some other project, but can't confirm as of today. 

### To reproduce

use powershell?
Arch. Wayland or X11. Hyprland or Plasma6.

### Screenshots

![image](https://github.com/user-attachments/assets/b74dc3c8-9382-427a-9113-b77b48ba1f29)

### Operating system

Linux

### Operating system and version

Arch

### Shell Version

PowerShell 7.4.6

### Current Warp version

v0.2024.12.18.08.02.stable_03

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Recent working Warp date

12/26/24

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this an issue only in Warp?

Yes, I confirmed that this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by alokedesai** [comment]
Hey @Phelsong, sorry for the issues here and thanks for filing.

Could you send us a copy of your powershell profile (with anything redacted if needed) so we can debug further? Are you able to repro with an empty profile?

**Comment by Phelsong** [comment]
Yes, I've confirmed with/without profile, Clean arch install, (pwsh) snap/bin/source-compiled.
![image](https://github.com/user-attachments/assets/109889d3-5aa3-448d-a619-380e8dce678a)

**Comment by acarl005** [comment]
We discovered a bug recently with usernames containing shell-ensafe characters, e.g. whitespace. Any chance it's that?

**Comment by Phelsong** [comment]
my user name is 5 characters, no letters or special characters

**Comment by acarl005** [comment]
Hi @Phelsong we've finally root-caused the issue. We've got a pull request to fix and it should be released next week.

**Comment by Phelsong** [comment]
fixed as of v0.2025.01.29.08.02.stable_03
